### ISSUE TITLE
Xposed Edge 5.5.1 not working

### ISSUE THREAD

**Issue by ChanceGPM** [taichi-framework/TaiChi#996]
As of now, Xposed Edge pro 5.5.1 is not compatible with ThaiChi and enabling the module does nothing. I would appreciate if you could add support for Xposed edge version 5.5.1. Thanks.

**Comment by tiann** [comment]
it cause bootloop on my oneplus 7pro Android 10, maybe it not stable

**Comment by tiann** [comment]
here is the crash stack:

```txt
2019-10-21 20:07:28.035 1510-1548/? E/XEdge: Caught throwable:
    java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.equals(java.lang.Object)' on a null object reference
        at android.hardware.camera2.CameraManager.getCameraCharacteristics(CameraManager.java:378)
        at com.jozein.xedge.b.W$d.a(Unknown Source:4)
        at com.jozein.xedge.b.W$d.g(Unknown Source:50)
        at com.jozein.xedge.b.W$d.<init>(Unknown Source:17)
        at com.jozein.xedge.b.W$d.<init>(Unknown Source:0)
        at com.jozein.xedge.b.W.a(Unknown Source:15)
        at com.jozein.xedge.xposed.Rb.M(Unknown Source:18)
        at com.jozein.xedge.xposed.Rb.i(Unknown Source:0)
        at com.jozein.xedge.xposed.Ma.a(Unknown Source:103)
        at com.jozein.xedge.xposed.Sc.run(Unknown Source:0)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:214)
        at android.os.HandlerThread.run(HandlerThread.java:67)
        at com.android.server.ServiceThread.run(ServiceThread.java:44)
        at com.android.server.UiThread.run(UiThread.java:43)
2019-10-21 20:07:28.445 1510-2126/? E/SensorService: FusionLight, invalid RGB data
```

**Comment by ChanceGPM** [comment]
Alright, hope you can fix it. Btw TV4PlayAdaway is a module request for me, it is not supported.

**Comment by tiann** [comment]
Please use version 5.5.3 Xposed Edge.

For TV4PlayAdaway, please open another issue, i will close this one.

**Comment by amosbird** [comment]
Hmm, when using 5.5.3 Xposed Edge pro on oneplus 7 pro, the left/right edge gesture has one limitation :  single press gesture fails to have any effect. Is it related to oneplus H2OS? I used to install Xposed Edge on MIUI with android 10 and it works as expected

**Comment by qwertsdf** [comment]
I have a Le Max 2 Android 10 with LineageOS. I tried Xposed Edge Pro 5.5.6. I found some problems. When I add an app to freezer on Xposed Edge Pro 5.5.6 and lauch it on freeser drawer, app suddenly stops itself. This is more rare on Xposed Edge Pro 5.2. But some app is same on 5.2. And other problem is keys section. When I activate "Back key" on Xposed Edge Pro (not depend on version), the tap back function for back button isn't working. Also previous app option on Xposed Edge Pro isn't working. Xposed Edge Pro should be checked with Android 10. Thank you.
